src/pages/Restaurants/RestaurantsPage.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import { CustomCardsSection, Navbar, Footer, RestaurantsHeader } from "../../components";
+import { RestaurantsData } from "../../data";
+import "./RestaurantsPage.scss"; // Import SCSS file
+
+const Restaurants = () => {
+  return (
+    <>
+      <Navbar />
+      <h2 className="restaurant-header">Restaurants</h2>
+      <RestaurantsHeader />
+      <CustomCardsSection cardsData={RestaurantsData} cardType={1} pageType={2} layoutDirection="vertical"/>
+      <div className="footer">
+      <div className='line'></div>
+      <Footer />
```

Comment:
If the footer will be on every page, you can simply place it once in the App.tsx file.